teamC/choojaeho/app.js

```diff
@@ -0,0 +1,43 @@
+const http = require ("http");
+const express = require ("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const dotenv = require("dotenv");
+dotenv.config()
+
+const {DataSource} = require('typeorm')
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+// myDataSource.initialize()
+//     .then(()=>{
+//         console.log("Data Source has been initialized!")
+//     })
+
+const app=express()
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan(''));
```

Comment:
morgan('')에는 빈 string이 들어가도록 의도하신걸까요? 해당부분에는 어떤 값들이 들어갈 수 있는지, 강의 영상과 어제 있었던 wrap-up에서도 다루었으니 채워서 작성해주세요~
```suggestion
app.use(morgan('someThing'));
```